packages/blockchain-components/src/utils/analytics/ClientAnalytics/const.ts

```diff
@@ -1,5 +1,6 @@
 export const eventNames = {
   TOKENGATE_COMPONENT_RENDERED: 'TOKENGATE_COMPONENT_RENDERED',
+  CONNECT_WALLET_COMPONENT_RENDERED: 'CONNECT_WALLET_COMPONENT_RENDERED',
```

Comment:
```suggestion
  CONNECT_WALLET_PROVIDER_MOUNTED: 'CONNECT_WALLET_PROVIDER_MOUNTED',
```

My thought here is that this is currently tracking when the `ConnectWalletProvider` renders. I might be wrong, but I'm thinking we might want more granular analytics here, such as:
- CONNECT_BUTTON_RENDERED
- CONNECT_MODAL_RENDERED
- CONNECTED_BUTTON_RENDERED

Probably splitting hairs here, but I thought it was something worth pointing out. What are your thoughts on this?

Reply:
Makes sense. For now, we will only track the provider but it makes sense to have the event name be more explicit about this

Reply:
Addressed here https://github.com/Shopify/blockchain-components/pull/82/commits/c09c8d2d16fa1aaddc56c69de1382530dc766560